docs/README.md

```diff
@@ -251,6 +251,43 @@ Note that users with [storefront permissions](#storefront-permissions) to edit q
 
 ![app-configuration](https://user-images.githubusercontent.com/6306265/163218059-903cfbcf-0077-45d2-9457-19ee0ba4d5c8.png)
 
+## **Metrics**
```

Comment:
I love that you documented this, but these docs automatically show up to our end-users (from the merchant or partner agencies), so they're not supposed to have internal information for our VTEX devs, only information for devs implementing this solution for their stores.

Cléber mentioned that we'll have an interna-docs repo to hold this kind of doc, so we should move this there.